Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -569,6 +569,7 @@ export type Props = $ReadOnly<{|
 |}>;
 
 type State = {|
+  contentKey: number, // TODO(macOS ISS#2323203)
```

Comment:
Is this really needed? Shouldn't the size change already trigger a re-render?

Reply:
It was not re-rendering without this.   Even a this.forceUpdate() wouldn't work.